ui/app/templates/clients/client.hbs

```diff
@@ -1,13 +1,131 @@
+        </div>
+      </div>
+      <div class="column is-centered is-minimum">
+        <button data-test-dismiss class="button is-danger" onclick={{action (mut eligibilityError) ""}}>Okay</button>
+      </div>
+    </div>
+  {{/if}}
+  {{#if stopDrainError}}
+    <div data-test-stop-drain-error class="columns">
+      <div class="column">
+        <div class="notification is-danger">
+          <h3 data-test-title class="title is-4">Stop Drain Error</h3>
+          <p data-test-message>{{stopDrainError}}</p>
+        </div>
+      </div>
+      <div class="column is-centered is-minimum">
+        <button data-test-dismiss class="button is-danger" onclick={{action (mut stopDrainError) ""}}>Okay</button>
+      </div>
+    </div>
+  {{/if}}
+  {{#if drainError}}
+    <div data-test-drain-error class="columns">
+      <div class="column">
+        <div class="notification is-danger">
+          <h3 data-test-title class="title is-4">Drain Error</h3>
+          <p data-test-message>{{drainError}}</p>
+        </div>
+      </div>
+      <div class="column is-centered is-minimum">
+        <button data-test-dismiss class="button is-danger" onclick={{action (mut drainError) ""}}>Okay</button>
+      </div>
+    </div>
+  {{/if}}
+  {{#if showDrainStoppedNotification}}
+    <div class="notification is-info">
+      <div data-test-drain-stopped-notification class="columns">
+        <div class="column">
+          <h3 data-test-title class="title is-4">Drain Stopped</h3>
+          <p data-test-message>The drain has been stopped and the node has been set to ineligible.</p>
+        </div>
+        <div class="column is-centered is-minimum">
+          <button data-test-dismiss class="button is-info" onclick={{action (mut showDrainStoppedNotification) false}}>Okay</button>
+        </div>
+      </div>
+    </div>
+  {{/if}}
+  {{#if showDrainUpdateNotification}}
+    <div class="notification is-info">
+      <div data-test-drain-updated-notification class="columns">
+        <div class="column">
+          <h3 data-test-title class="title is-4">Drain Updated</h3>
+          <p data-test-message>The new drain specification has been applied.</p>
+        </div>
+        <div class="column is-centered is-minimum">
+          <button data-test-dismiss class="button is-info" onclick={{action (mut showDrainUpdateNotification) false}}>Okay</button>
+        </div>
+      </div>
+    </div>
+  {{/if}}
+  {{#if showDrainNotification}}
+    <div class="notification is-info">
+      <div data-test-drain-complete-notification class="columns">
+        <div class="column">
+          <h3 data-test-title class="title is-4">Drain Complete</h3>
+          <p data-test-message>Allocations have been drained and the node has been set to ineligible.</p>
+        </div>
+        <div class="column is-centered is-minimum">
+          <button data-test-dimiss class="button is-info" onclick={{action (mut showDrainNotification) false}}>Okay</button>
```

Comment:
I see that there’s a `clickable` `dismiss` in the page object but I didn’t see any use of it in any tests, is it an oversight? I did try it out with this particular notification and it could be overkill to have assertions that dismissal works for every variation, but maybe the major one could have one? Or maybe that `clickable` can just be removed.

Reply:
It's used a few times in the acceptance tests.

https://github.com/hashicorp/nomad/blob/73c9ce41c0c316e2a7c22e0e03f63223e678bca0/ui/tests/acceptance/client-detail-test.js#L811-L812

Github may have auto-hid the diff and tricked you if you were trying to cmd+f for it.

Reply:
oops, that’s entirely true, thanks 😯